src/mass/index.ts

```diff
@@ -18,6 +18,8 @@ export function withValueType<NumberType>(arithmetic: Arithmetic<NumberType>) {
     static kilograms = makeUnit('kg', dimension.Mass);
 
     static grams = WithValueType.kilograms.times(1e-3).withSymbol('g');
+
+    static pounds = WithValueType.kilogram(4.535924e-1).withSymbol('lb');
```

Comment:
Can you please use 0.45359237 which is the official definition of the pound?
https://en.wikipedia.org/wiki/International_yard_and_pound

Reply:
Done. I don't think that it matters (it just rounds it) but thanks for your review :)